examples/command/parse_no_execute.ts

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env -S deno run
+
+import { Command } from '../../command/command.ts';
+import { CompletionsCommand } from '../../command/completions/completions_command.ts';
+import { HelpCommand } from '../../command/help/help_command.ts';
+
+function getCommandFullname(command: Command<any> | undefined): string {
```

Comment:
Is there a way to avoid using `Command<any>` for this sort of thing. I've had this problem in my own code using cliffy as well.